src/api/PersistentSearchApi.test.js

```diff
@@ -57,6 +58,20 @@ describe(PERSISTENT_SEARCH_API, () => {
           },
         },
       },
+      updatePersistentSearchExpiration: {
+        '': { params: { optional: [false], valid: [MOCK_PS_ID, MOCK_EXPIRATION] } },
+        '(updatePersistentSearchExpiration)': {
```

Comment:
The intent with this syntax here is to capture the parameters being passed. Please update to something like:

```
'(id, expiration)'
```